ghzdac/correction.py

```diff
@@ -1165,10 +1188,17 @@ def DACifyFT(self, signal, t0=0, n=8192, offset=0, nfft=None, loop=False,
                 #decay times:
                 # add to qubit registry the following keys:
                 # settlingAmplitudes=[-0.05]  #relative amplitude
-                # settlingRates = [0.012]    #rate is in GHz, (1/ns)
+                # settlingRates = [0.01]    #rate is in GHz, (1/ns)
                 if np.alen(decayRates):
-                    freqs = 2j*np.pi*freqs
```

Comment:
I was referring just to the subexpression `2j*np.pi*freqs`, which previously was being computed once and then used twice in the settling response function. The way you've written it, this subexpression would be reevaluated several times in both the settling and reflection terms. Not a big difference either way, though.